troubleshooting/troubleshooting-ios-cannot-load-iframe.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Cannot Load iframe on iOS devices when using Cordova 5.0.0 or later
+description: Cannot load iframe on iOS devices with Cordova 5.0.0 and 5.2.0.
+type: troubleshooting
+page_title: Cordova 5.0.0 or later apps do not load iframe on iOS
+slug: iframe-not-loading-ios
+position: 
+tags: appbuilder, ios, cordova 
+teampulseid: 
+ticketid: 1060248
+pitsid: 
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>	
+  </tr>
+
+  <tr>
+    <td>Mobile OS</td>
+    <td>iOS</td>	
+  </tr>
+  <tr>
+    <td>Cordova framework version</td>
+    <td>Cordova 5.0.0 and above</td>	
```

Comment:
http://docs.telerik.com/platform/knowledge-base/troubleshooting/troubleshooting-additional-digit-appended-to-apk